.gitignore

```diff
@@ -37,4 +38,4 @@ yarn-error.log*
 .vscode/*
 # but allow files with default or shared to be included as example setups
 !.vscode/*.default*
-!.vscode/*.shared*
\ No newline at end of file
+!.vscode/*.shared*package.json
```

Comment:
This doesn't seem correct. I would not expect there to be a .sharedpackage.json file as well it removes the .shared folder

 what were you intending to do with this edit?